### ISSUE TITLE
"Call to undefined function env()" on Laravel 4.2 project

### ISSUE THREAD

**Issue by bcalik** [itsgoingd/clockwork#237]
This is what I get after upgrading clockwork from v2.0.3 to v2.1, on Laravel 4.2.

<img width="1494" alt="screen shot 2018-02-26 at 13 31 30" src="https://user-images.githubusercontent.com/9222368/36665727-61464dc6-1af9-11e8-81ba-997a851b2091.png">

**Comment by bcalik** [comment]
Looks like you are going to drop Laravel 4.2 support in future versions. Maybe you should move this change to the next version.

Anyway I added this helper from Laravel 5 to my project for now:

```
if (! function_exists('env')) {
    /**
     * Gets the value of an environment variable. Supports boolean, empty and null.
     *
     * @param  string  $key
     * @param  mixed   $default
     * @return mixed
     */
    function env($key, $default = null)
    {
        $value = getenv($key);

        if ($value === false) {
            return value($default);
        }

        switch (strtolower($value)) {
            case 'true':
            case '(true)':
                return true;
            case 'false':
            case '(false)':
                return false;
            case 'empty':
            case '(empty)':
                return '';
            case 'null':
            case '(null)':
                return;
        }

        if (strlen($value) > 1 && Str::startsWith($value, '"') && Str::endsWith($value, '"')) {
            return substr($value, 1, -1);
        }

        return $value;
    }
}
```

**Comment by itsgoingd** [comment]
Oops, tagged 2.1.1 with a fix for this issue, let me know if it works for you.

**Comment by bcalik** [comment]
@itsgoingd It works. Thanks.

So v2.1.1 is the last version supporting Laravel 4, right?

**Comment by itsgoingd** [comment]
Hey, we are dropping Laravel 4 support in 3.0, so you can safely require `^2.0` in your project to get the latest 2.x version (2.2 was just released today).

**Comment by bcalik** [comment]
Ok, I get this error on 2.2 :) @itsgoingd 

![screen shot 2018-03-09 at 02 45 31](https://user-images.githubusercontent.com/9222368/37182996-f48f7ec4-2344-11e8-9a2f-cc04c75b4a9d.png)

**Comment by itsgoingd** [comment]
Fixed and tagged 2.2.1, should be available shortly, sorry for the inconvenience.

**Comment by bcalik** [comment]
Thanks for the great work! @itsgoingd
